catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -757,12 +760,22 @@ public void setQueryTypeNamespace(String queryTypeNamespace) {
 
     @Override
     public boolean isAvailable() {
+
```

Comment:
❓ Why this change at all? And even if necessary, it seems unrelated to the work at hand.

Reply:
Most likely a typo or overzealous auto-format.